.github/workflows/im-reusable-finish-build-workflow.yml

```diff
@@ -66,10 +66,19 @@ on:
         description: Additional PR comment content aside from workflow run and next version.
         required: false
         type: string
+      send-teams-notification:
+        description: Flag indicating whether to send a notification to the team's channel.  Expected `true` or `false`.
+        required: false
+        type: boolean
+        default: true
```

Comment:
Would having this default to true just throw an error for those teams that don't want to deploy to their teams channel, so they would have to specify `false`?

It's a negative feed back opt-in.